docs/release-notes/2.25.0.md

```diff
@@ -0,0 +1,94 @@
+- [PMM-7079](https://jira.percona.com/browse/PMM-7079): Integrated Alerting: New 'information' icon to give additional details about Alerts without cluttering screen
+- [PMM-8259](https://jira.percona.com/browse/PMM-8259): Better clarification of error messages in `pmm-admin` when PMM server can't be unregistered 
+- [PMM-8770](https://jira.percona.com/browse/PMM-8770): Enable safe Collection metrics collectors for MongoDB
+- [PMM-8972](https://jira.percona.com/browse/PMM-8972): Add ability to specify custom base path to exporters and tools using `pmm-admin` command
+- [PMM-8282](https://jira.percona.com/browse/PMM-8282): Improved messaging for TLS option when adding Remote instances in PMM over UI 
+
+## Bugs Fixed
+
+- [PMM-9169](https://jira.percona.com/browse/PMM-9169): Security Advisor Checks are not working for MongoDB instances
+- [PMM-8982](https://jira.percona.com/browse/PMM-8982): Backup Management: User is not able to see MongoDB backup logs if backup was taken on older version of pmm-server 
+- [PMM-9157](https://jira.percona.com/browse/PMM-9157): Dashboards: Changing the timezone on dashboards does not persist navigation
+- [PMM-7116](https://jira.percona.com/browse/PMM-7116): Dashboards: Incorrect STARTUP state on MongoDB ReplSet Summary dashboard
+- [PMM-8993](https://jira.percona.com/browse/PMM-8993): Integrated Alerting: Sending email using Gmail fails
+- [PMM-7802](https://jira.percona.com/browse/PMM-7802): PMM can't monitor MongoDB arbiter nodes (Thanks to Artem Meshcheryakov for reporting this issue)
+- [PMM-6937](https://jira.percona.com/browse/PMM-6937): Can't add PostgreSQL instances to PMM without `postgres` DB in PostgreSQL server (Thanks to Daniel Kowalewski for reporting this issue)
+- [PMM-7447](https://jira.percona.com/browse/PMM-7447): Can't add into PMM instances of PostgreSQL with SCRAM-SHA-256 authentication
+- [PMM-9085](https://jira.percona.com/browse/PMM-9085): PMM Server crashes after upgrading to 2.22 every 4 hours
+- [PMM-9156](https://jira.percona.com/browse/PMM-9156): `pmm-agent` paths-base option not working for pmm2-client binary installation in PMM 2.23.0
+- [PMM-8461](https://jira.percona.com/browse/PMM-8461): DBaaS: Confusing error when accessing DBaaS pages when it's disabled
+- [PMM-8110](https://jira.percona.com/browse/PMM-8110): DBaaS: Registering K8s cluster with operators already installed can cause error
+- [PMM-8694](https://jira.percona.com/browse/PMM-8694): Query Analytics: URLs in Query Analytics with a selected query and a timestamp range does not select the query
+- [PMM-9227](https://jira.percona.com/browse/PMM-9227): Pagination Reset on QAN after Time Range change doesn't work, results in wrong results
+- [PMM-9298](https://jira.percona.com/browse/PMM-9298): PMM AMI image in 2.24.0 has only 8GB space for data and Volume Size Check fails while upgrading to 2.25.0

```

Comment:
## Known issue
After connecting PMM to Percona Platform, PMM occasionally shows a false permission issue notification, incorrectly suggesting that the connection could not be established due to missing permissions. Reload the page to remove the incorrect notification and confirm the connection.